src/dialog/index.css

```diff
@@ -59,7 +82,7 @@
 
 
 .jp-Dialog-okButton {
-  background: #2196F3;
+  background: var(--jp-brand-color1);
   color: white;
```

Comment:
`white` might not work with all themes, so it should probably be the accent color against the brand color.